tests/simulator/test_diagnostic_timestamps.py

```
@@ -101,62 +101,43 @@ def tearDown(self):
                 simInput = simArgs.copy()
-                diag_outputs = f"phare_outputs_hierarchy_timestamp_cadence_{dim}_{self.ddt_test_id()}_{i}"
+                diag_outputs = f"phare_outputs_hierarchy_timestamp_cadence_{dim}_{self.ddt_test_id()}_{diag_cadence}"
                 simInput["diag_options"]["options"]["dir"] = diag_outputs
                 simInput["time_step_nbr"] = time_step_nbr
 
                 ph.global_vars.sim = None
                 simulation = ph.Simulation(**simInput)
                 setup_model(10)
 
-                timestamps = np.arange(0, final_time, time_step * i)[trailing:]
+                timestamps = np.arange(0, final_time, time_step * i)[skip_init:]
```

Ruff (0.8.2)

Undefined name `i`

(F821)